src/registrar/views/domain.py

```diff
@@ -118,6 +119,7 @@ def post(self, request, *args, **kwargs):
         if form.is_valid():
             return self.form_valid(form)
         else:
+            logger.debug(f"Form errors: {form.errors}")
```

Comment:
Not strictly necessary, but I found this super useful while debugging this PR so thought I might leave it in.